infra/main.parameters.json

```diff
@@ -68,6 +68,9 @@
     "useApplicationInsights": {
       "value": "${AZURE_USE_APPLICATION_INSIGHTS=false}"
     },
+    "allowedOrigin": {
+      "value": "${ALLOWED_ORIGIN}"
+    },
```

Comment:
Output is ALLOWED_ORIGINS and value is ALLOWED_ORIGIN. Is that a typo?

Reply:
No, that's wanted: `ALLOWED_ORIGIN` is one additional frontend url (for future separate deployments), and output `ALLOWED_ORIGINS` is the list of ALL allowed origins, including the optional input one and our frontend